hphp/runtime/vm/jit/vasm-check.cpp

```diff
@@ -60,14 +60,16 @@ checkSSA(const Vunit& unit, const jit::vector<Vlabel>& blocks) {
     }
     for (auto const& inst : unit.blocks[b].code) {
       visitUses(unit, inst, [&](Vreg v) {
-        assert_flog(v.isValid(), "invalid vreg used in B{}\n{}",
+        assert_flog(v.isValid() || inst.op == Vinstr::pushp,
+		    "invalid vreg used in B{}\n{}",
```

Comment:
Here and below are two places where I had to adjust the SSA checks to allow `InvalidReg` for uses and defs for `pushp` and `popp`.

I don't really like this, but I tried it out. Personally, I prefer to just push and pop the same registers, but I imagine that check SSA could fail too if we're def'ing the same register twice in one instruction.